__tests__/app.test.js

```diff
@@ -38,3 +38,27 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: returns an array of article objects with the correct properties & sorted by date descending ", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Array);
+        expect(body).not.toHaveLength(0);
+        expect(body).toBeSortedBy("created_at", { descending: true });
+        body.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(String),
+          });
```

Comment:
if you wanted to have this comment_count value stored as a number instead, you can coerce the value into an integer in your sql statement by saying COUNT(column here..)**::INT**